docs/sdks/stash/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
+
+As of 2023-07-15 plugin version:
+* iOS V 0.2.0
+* Android V 0.4.0
+
+## Introduction
+
+The UID2 GMA Plugin is pre-release software and should be used as such.
+
+The UID2 GMA Plugin enables apps that use Google Mobile Ad (GMA) SDK to
```

Comment:
Refer to it as _The UID2 Google Mobile Ads (GMA) Plugin_ once here and leave all other mentioning to _The UID2 GMA Plugin_ just to make it clear

Reply:
Done for stashed external file.